packages/core/src/components/TextField/types.ts

```diff
@@ -13,6 +13,7 @@ export interface TextFieldProps
   disabled?: boolean;
   rows?: number;
   endElement?: React.ReactElement;
+  startElement?: React.ReactElement;
```

Comment:
I suggest extending this type from `InputProps`. What do you think?
TextField is a wrapper over Input. And currently, TextField has duplicate type definitions, which complicates support.